src/commands/deleteVirtualMachine/ConfirmDeleteStep.ts

```diff
@@ -0,0 +1,33 @@
+/*---------------------------------------------------------------------------------------------
+*  Copyright (c) Microsoft Corporation. All rights reserved.
+*  Licensed under the MIT License. See License.txt in the project root for license information.
+*--------------------------------------------------------------------------------------------*/
+
+import { AzureWizardPromptStep, DialogResponses, nonNullProp } from "@microsoft/vscode-azext-utils";
+import { virtualMachineLabel } from "../../constants";
+import { localize } from "../../localize";
+import { IDeleteChildImplContext } from "./deleteConstants";
+
+export class ConfirmDeleteStep extends AzureWizardPromptStep<IDeleteChildImplContext> {
```

Comment:
Can we use the shared `DeleteConfirmationStep` and just set the message based on the resource count?

Reply:
Oh, I see now--the `SelectResourcesToDelete` step can choose one or many, and there's not an opportunity to put code inbetween the execution of that step and `ConfirmDeleteStep`. Nevermind then!